node_modules/col-activities/lib/notifications/daily.js

```diff
@@ -0,0 +1,548 @@
+/**
+ * Copyright ©2015. The Regents of the University of California (Regents). All Rights Reserved.
```

Comment:
But maybe after the migration script PR is merged, which takes care of a large number of them